sdk/communication/communication-common/test/identifierModelSerializer.spec.ts

```diff
@@ -83,27 +85,57 @@ describe("Identifier model serializer", () => {
     assertSerialize(
       { microsoftTeamsUserId: "37691ec4-57fb-4c0f-ae31-32791610cb14", isAnonymous: true },
       {
+        rawId: "8:teamsvisitor:37691ec4-57fb-4c0f-ae31-32791610cb14",
         microsoftTeamsUser: {
           userId: "37691ec4-57fb-4c0f-ae31-32791610cb14",
           isAnonymous: true,
           cloud: "public",
         },
       }
     );
+    assertSerialize(
+      { microsoftTeamsUserId: "37691ec4-57fb-4c0f-ae31-32791610cb14", rawId: "override" },
```

Comment:
What is this case where `rawId` would be this override string?

Reply:
![image](https://user-images.githubusercontent.com/82062616/166070900-8e88879f-5286-44d0-b68b-9e5fe099c6e9.png)

looks like a legit example.

Reply:
Unfortunately, it is a legitimate example and can happen. The first 5000 Skype users still have a MRI format that includes the tenant id.